### ISSUE TITLE
A11y: upload progess bar

### ISSUE THREAD

**Issue by csutorasr** [box/box-ui-elements#2612]
**Environment:**

- Elements version:

**Desktop (please complete the following information):**

- OS: Windows 10
- Browser: Chrome
- Version 91

**Steps to reproduce the problem:**

Upload a file with screen reader. The progressbar is not announced to the users with its progress.

**What is the expected behavior?**

Announce the progress bar values.

**Additional context**

Accoding to Mozila on [progressbar role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_progressbar_role#possible_effects_on_user_agents_and_assistive_technology) the changes should be announced if the role and the values are set correctly.
